src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -923,3 +923,93 @@ BEGIN
     ORDER BY Sequence DESC
 END
 GO
+
+/*************************************************************
+    SQL VERSION 2
+*************************************************************/
+/*************************************************************
+    TABLES
+*************************************************************/
+
+/*************************************************************
+    Custom Tag Table
+    Stores added custom tags
+    TagPath is represented without any delimiters and each level takes 8 bytes
+    TagLevel can be 0, 1 or 2 to represent Instance, Series or Study level
+**************************************************************/
+CREATE TABLE dbo.CustomTag (
+    TagKey                  BIGINT               NOT NULL, --PK
+    TagPath                 VARCHAR(64)          NOT NULL,
+    TagVR                   VARCHAR(2)           NOT NULL,
+    TagLevel                TINYINT              NOT NULL,
+    TagStatus               TINYINT              NOT NULL,
+)
+
+CREATE UNIQUE CLUSTERED INDEX IXC_CustomTag ON dbo.CustomTag
+(
+    TagKey
+)
+
+CREATE NONCLUSTERED INDEX IX_CustomTag_TagPath ON dbo.CustomTag
```

Comment:
can we do a unique on this this index?

Reply:
I guess Sneha will do this, but it's also good to change here. Updated.